scipy/stats/tests/test_stats.py

```diff
@@ -3885,6 +3886,17 @@ def test_gh11184_bigger(self):
         self._testOne(x, y, 'greater', 0.10597913208679133, 2.7947433906389253e-41, mode='asymp')
         self._testOne(x, y, 'less', 0.09658002199780022, 2.7947433906389253e-41, mode='asymp')
 
+    @pytest.mark.slow
```

Comment:
This is quite fast on my machine. Is this marked slow only because Pythran is still optional and it's slow in pure-Python mode?

Reply:
1.2 sec on my machine. Probably 1.5-2x slower in CI, but I think that's okay. So I'd say it's fine to remove `@slow` unless it's way more than that in your notebook.

Reply:
@rgommers What should we do about this one? It's the slowest test on a few builds, e.g.:
```
============================ slowest 10 durations =============================
275.92s call     stats/tests/test_stats.py::TestKSTwoSamples::test_gh12999
```

Reply:
Yes, it may indeed warrant a closer look. That test has been problematic before, +1 for moving to `xslow`.